userguide/one_shot_object_detection/README.md

```diff
@@ -1,41 +1,40 @@
 # One-Shot Object Detection
 
-*Note: The One Shot Object Detector is currently in beta.*
-
-One-Shot object detection (OSOD) is the task of detecting an object from as few as **one** example per category. Unlike the [Object Detector](https://apple.github.io/turicreate/docs/userguide/object_detection) which requires many varied examples of objects in the real world, the One-Shot detector only requires a single canonical example of an object to train the classifier on and perform predictions for that category in the wild. It is best suited for 2D objects or when there isn't much expected variance in the examples, such as playing cards, logos, road signs, and clapperboards. One-Shot detector is not suitable for 3D objects such as faces, animals, and cars which is a task better suited for the regular [Object Detector](https://apple.github.io/turicreate/docs/userguide/object_detection). 
-
-
+One-Shot object detection (OSOD) is the task of detecting an object from as little as **one** example per category. Unlike the [Object Detector](https://apple.github.io/turicreate/docs/userguide/object_detection) which requires many varied examples of objects in the real world, the One-Shot Object Detector requires a very small (sometimes even just one) canonical example of the object. The One-shot Object Detector is best suited for two-dimensional objects that have some regularity in the wild. Examples include road signs, logos, playing cards, and clapperboards. The One-Shot Object Detector is not suitable for three-dimensional objects like faces, animals, and cars -- such objects are better suited for the classical [Object Detector](https://apple.github.io/turicreate/docs/userguide/object_detection). 
 
 ## Introductory Example
 
 In this example, our goal is to **localize** instances of **stop signs** in images. To train a model for this application, we supply a single image of a stop sign as a starter image.
 
-Given a starter image (*left*), a one-shot detector trained on this starter image will produce instance predictions on a test image (*right*) which may look like this:
+Given a starter image (*left*), a one-shot object detector trained on this starter image will produce instance predictions on a test image (*right*) which may look like this:
 
 ![Stop Sign vector and prediction images](images/stop_sign_sample.jpg) 
 
+Download the starter image to get started from [here](images/stop_sign_starter.png).
```

Comment:
I think it pretty bad form to hyper link the word `here`. I think this is much better
```
Download the [stop_sign_starter.png](images/stop_sign_starter.png) image. Also Download [stop_sing_photo1.png](images/stop_sign_test1.jpg) and [stop_sing_photo2.png](images/stop_sign_test2.jpg)